src/pyclass_init.rs

```diff
@@ -114,14 +114,27 @@ impl<T: PyClass> PyClassInitializer<T> {
         PyClassInitializer::new(subclass_value, self)
     }
 
-    // Create a new PyCell + initialize it
-    #[doc(hidden)]
-    pub unsafe fn create_cell(self, py: Python) -> PyResult<*mut PyCell<T>>
+    // Create a new PyCell and initialize it.
+    pub(crate) unsafe fn create_cell(self, py: Python) -> PyResult<*mut PyCell<T>>
```

Comment:
This function doesn't actually look unsafe to me? From the signature, worst case seems to be memory leak which is sound.

```suggestion
    // Create a new PyCell and initialize it.
    pub(crate) fn create_cell(self, py: Python) -> PyResult<*mut PyCell<T>>
```

Reply:
:+1: